patches/disable-builtin-ext-update.diff

```diff
@@ -18,14 +18,3 @@ Index: code-server/lib/vscode/src/vs/workbench/contrib/extensions/browser/extens
  			if (!this.local.preRelease && this.gallery.properties.isPreReleaseVersion) {
  				return false;
  			}
-@@ -1237,6 +1241,10 @@ export class ExtensionsWorkbenchService 
- 				// Skip if check updates only for builtin extensions and current extension is not builtin.
- 				continue;
- 			}
-+			if (installed.type !== ExtensionType.User) {
-+				// Never update builtin extensions.
-+				continue;
-+			}
```

Comment:
Unsure if we still need this. See 6751c827cc317184d196db962906377928f0b172

cc @code-asher

Reply:
I think you are right!  One way to see is if on load Code downloads a bunch of builtin extensions into the extensions dir.